client/src/components/NavBar/NavBar.tsx

```diff
@@ -26,13 +38,20 @@ const NavBar = (): JSX.Element => {
   if (loggedInUser === undefined) return <CircularProgress />;
 
   return (
-    <AppBar className={classes.appbar} position="absolute">
-      <CssBaseline />
-      <ToolBar className={classes.toolbar}>
-        <img src={logo} alt="logo" />
-        {loggedInUser ? <LoggedInBar /> : <LoggedOutBar />}
-      </ToolBar>
-    </AppBar>
+    <Grid>
+      <AppBar
+        elevation={history.location.pathname === '/' ? 0 : 6}
+        className={isLandingPage ? classes.landingPageBar : classes.appbar}
+        position="absolute"
+      >
+        <CssBaseline />
```

Comment:
Please remove the `CssBaseline`component as is a global reset and should be included only once, ideally in the App.tsx

Reply:
I used the code from another component and forgot to check it. Done.